### PR TITLE
Allow to edit the analysis service sort keys in the services listing

### PR DESCRIPTION
**Pull request by ramonski**
## Description of the issue/feature this PR addresses

This PR allows to edit the analysis service sort keys in the services listing:

Analysis Services — SENAITE LIMS 2022-09-13 4 PM-17-48

## Current behavior before PR

Service sort keys could only be edited in the edit view

## Desired behavior after PR is merged

Service sort keys can be edited inside the services listing

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: src/bika/lims/browser/analyses/view.py [hunk 1]
```diff
@@ -885,7 +885,7 @@ def _folder_item_result(self, analysis_brain, item):
 
             else:
                 item["result_type"] = "numeric"
-                item["help"]["result"] = _(
+                item["help"]["Result"] = _(
```
Comment (ramonski) [inline src/bika/lims/browser/analyses/view.py:899]: Compatibility for https://github.com/senaite/senaite.app.listing/pull/85

**Review by xispa — CHANGES_REQUESTED** [review]
The Unit is displayed next to the SortKey input field, which is confusing.

On another note, how is the Data Manager for AnalysisService related with this "SortKey" functionality?. I am not sure if I am missing something.

**Comment by ramonski** [comment]
> The Unit is displayed next to the SortKey input field, which is confusing.
> 
> On another note, how is the Data Manager for AnalysisService related with this "SortKey" functionality?. I am not sure if I am missing something.

Ups, apologies. Going to change that.

The datamanagers are called from here:
https://github.com/senaite/senaite.app.listing/blob/2.x/src/senaite/app/listing/ajax.py#L392

Probably better to rename them one day into `IAjaxFieldManager`

**Comment by ramonski** [comment]
Oh, the behavior with the unit comes actually from here:
https://github.com/senaite/senaite.app.listing/blob/2.x/webpack/app/components/TableCell.coffee#L420

Probably because numeric fields used to be either result or interim fields...

**Comment by ramonski** [comment]
Implicit Unit injection was removed in https://github.com/senaite/senaite.app.listing/pull/86 and explicitly added in https://github.com/senaite/senaite.core/pull/2134

**Review by xispa — APPROVED** [review]

**Pull request merged**